verl/workers/actor/dp_actor.py

```diff
@@ -74,6 +124,318 @@ def __init__(self, config, actor_module: nn.Module, actor_optimizer: torch.optim
         )
         self.device_name = get_device_name()
 
+        # Note(vermouth1992): we move the device mesh constructor into the DataParallelActor
+        # so that DataParallelActor can be instantiated solely base on config. We keep compatible with old approach
+        # that constructing model and optimizer, when pass them as arguments. We will remove the compatibility in v0.6
+        self.rank = torch.distributed.get_rank()
+        self.inference_only = inference_only
+        self._lora_rank = self.config.model.get("lora_rank", 0)
+        self._is_lora = self._lora_rank > 0
+        # build device mesh for FSDP
+        world_size = torch.distributed.get_world_size()
+        # TODO(sgm): support FSDP hybrid shard for larger model
+        self.device_mesh = create_device_mesh(world_size=world_size, fsdp_size=self.config.fsdp_config.fsdp_size)
```

Comment:
the device mesh was still in https://github.com/volcengine/verl/blob/0eae03ae2bb369012c230edcb44de244e520e12b/verl/workers/fsdp_workers.py#L90